cmd/wio/config/defaults.go

```diff
@@ -1,23 +1,29 @@
 package config
 
-type avrDefaults struct {
-    Ide           string
-    Framework     string
+import "wio/cmd/wio/constants"
+
+type defaults struct {
```

Comment:
defaults doesn't even need to be a struct, something like

```
package defaults

var Version = "0.0.1"
var AppSource = "src"
var PkgSource = "tests"
// ...
```

Reply:
that is true I can fix that